lab-mongoose-movies/routes/index.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express');
+const router  = express.Router();
+
+router.get('/', (req, res, next) => {
+  res.render('celebrities');
```

Comment:
You are rendering the view before the DB logic, so the function ends here, and you are not really fetching from the database. This line should be removed